.circleci/download_llvm_raw.sh

```diff
@@ -4,7 +4,7 @@
 function download_llvm_raw_archive() {
   # Extract the xla pinned tensorflow/third_party/llvm version,
   # avoid doing this manually.
-  git clone --recursive --quiet https://github.com/pytorch/xla.git
+  git clone --recursive --quiet --branch r2.0 https://github.com/pytorch/xla.git
```

Comment:
I wonder if it's necessary and also if we should do the same for upstream pytorch.

Reply:
nvm, i saw our chat.